.github/workflows/release-v2.yml

```diff
@@ -10,7 +10,6 @@ on:
       publish:
         description: 'Publish docker images and JARs to OSSRH?'
         required: true
-        default: false
```

Comment:
Would this change anything? Is the idea that default value from workflow_dispatch would somehow have "leaked" to on-push triggering? And that now default would be `null`.
From discussions it seemed more likely that instead it's just that for the most part `null` and `false` behave the same and the default on-push uses might just be that of "falsy" (undefined/null, but something that coerces to `false` in boolean context).

In a way, reversing the logic could solve this (as I mentioned in discussion). But alternatively, maybe on-push trigger could pass explicit `true` for this value -- if `inputs` entries can be passed by triggers other than "workflow_dispatch".